src/main/java/org/spongepowered/common/mixin/core/village/MerchantOfferMixin.java

```diff
@@ -24,26 +24,28 @@
  */
 package org.spongepowered.common.mixin.core.village;
 
-import net.minecraft.village.MerchantRecipe;
+import net.minecraft.item.MerchantOffer;
 import org.spongepowered.api.item.inventory.ItemStack;
 import org.spongepowered.api.item.inventory.ItemStackComparators;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
 
 import javax.annotation.Nullable;
 
-@Mixin(MerchantRecipe.class)
-public abstract class MerchantRecipeMixin {
+@Mixin(MerchantOffer.class)
```

Comment:
Target changed, should go into...

core.item

Reply:
Well that one is a duplicate from the conflicts so its now gone.